allennlp_models/pair_classification/models/bimpm.py

```diff
@@ -226,23 +225,33 @@ def add_matching_result(matcher, encoded_premise, encoded_hypothesis):
 
         return output_dict
 
+    @overrides
+    def get_metrics(self, reset: bool = False) -> Dict[str, float]:
+        return {
+            metric_name: metric.get_metric(reset) for metric_name, metric in self.metrics.items()
+        }
+
     @overrides
     def make_output_human_readable(
         self, output_dict: Dict[str, torch.Tensor]
     ) -> Dict[str, torch.Tensor]:
         """
-        Converts indices to string labels, and adds a `"label"` key to the result.
+        Does a simple argmax over the probabilities, converts index to string label, and
+        add `"label"` key to the dictionary with the result.
         """
-        predictions = output_dict["probs"].cpu().data.numpy()
-        argmax_indices = numpy.argmax(predictions, axis=-1)
-        labels = [self.vocab.get_token_from_index(x, namespace="labels") for x in argmax_indices]
-        output_dict["label"] = labels
+        predictions = output_dict["label_probs"]
```

Comment:
not sure what the right thing to do here is---bimpm outputs probs under the `probs` key, but the other models use `label_probs`. should we standardize bimpm and change `probs` to `label_probs` here, too?

Reply:
Yes, I think that's the best change.